templates/taxi/driver_detail.html

```diff
@@ -8,6 +8,8 @@ <h1>
   <p><strong>First name:</strong> {{ driver.first_name }}</p>
   <p><strong>Last name:</strong> {{ driver.last_name }}</p>
   <p><strong>License number:</strong> {{ driver.license_number }}</p>
+  <a href="{% url 'taxi:driver-update' pk=driver.pk %}" class="btn btn-primary">Update license</a>
+  <a href="{% url 'taxi:driver-delete' pk=driver.pk %}" class="btn btn-danger">Delete license</a>
```

Comment:
The 'Delete license' button is misleading because it suggests deleting only the license, whereas it actually deletes the entire driver. Consider renaming the button to 'Delete driver' to accurately reflect the action being performed.